lib/wasm/src/code_translator.rs

```diff
@@ -309,14 +308,13 @@ pub fn translate_operator<FE: FuncEnvironment + ?Sized>(
                     data.push_entry(branch_ebb);
                 }
```

Comment:
The default edge can also be critical, so we should also add a `dest_ebb_sequence` entry for the default edge here too.

Reply:
And done!